client/src/modules/tags/tags.html

```diff
@@ -0,0 +1,40 @@
+<div class="flex-header">
+    <div class="bhima-title">
+      
+      <ol class="headercrumb">
+        <li class="static" translate>TREE.ADMIN</li>
+        <li class="title" translate>TREE.TAGS</li>
+      </ol>
+  
+      <div class="toolbar">
+        <div class="toolbar-item">
+          <button class="btn btn-default" ng-click="TagsCtrl.createUpdateTagsModal()" data-method="create">
+            <span class="fa fa-plus"></span>
+            <span translate class="text-capitalize">FORM.LABELS.ADD</span>
```

Comment:
If you include `class="hidden-xs"` next to the `text-capitalize` class, the text will disappear on smaller screens, which is nice.